app-editors/emacs/emacs-30.1-r1.ebuild

```diff
@@ -463,7 +435,6 @@ src_test() {
 	# Redirect GnuPG's sockets, in order not to exceed the 108 char limit
 	# for socket paths on Linux.
 	mkdir "${T}"/gpg || die
-	local f
```

Comment:
Why would we want to assign `f` in global scope? The local declaration was correct.